.yamllint.yaml

```diff
@@ -0,0 +1,49 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+
+ignore:
+  - deploy/helm-chart/templates/
```

Comment:
Would it cause an error to add `deploy/helm-chart/crds/` to this since it doesn't exist yet (but likely will in the near future).

Reply:
There's no error, seems no harm to add it 😄